matminer/datasets/tests/test_convenience_loaders.py

```diff
@@ -0,0 +1,181 @@
+
+    # Skip for circleCI efficiency
+    @unittest.skip
+    def test_load_glass_ternary_landolt(self):
+        df = load_glass_ternary_landolt()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df), 6118)
+
+        df = load_glass_ternary_landolt(unique_composition=False)
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df), 7191)
+
+        df = load_glass_ternary_landolt(processing="meltspin")
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df), 5736)
+
+        df = load_glass_ternary_landolt(processing="sputtering")
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df), 403)
+
+        with self.assertRaises(ValueError):
+            df = load_glass_ternary_landolt(processing="spittering")
+
+    def test_load_phonon_dielectric_mp(self):
+        df = load_phonon_dielectric_mp()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    def test_load_glass_binary(self):
+        df = load_glass_binary()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    def test_load_m2ax(self):
+        df = load_m2ax()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    def test_load_expt_gap(self):
+        df = load_expt_gap()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    def test_load_expt_formation_enthalpy(self):
+        df = load_expt_formation_enthalpy()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    # Skip due to memory usage
+    @unittest.skip
+    def test_load_mp(self):
+        df = load_mp()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df.columns), 9)
+
+        df = load_mp(include_structures=True)
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df.columns), 11)
+
+    def test_load_wolverton_oxides(self):
+        df = load_wolverton_oxides()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    def test_load_heusler_magnetic(self):
+        df = load_heusler_magnetic()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    def test_load_steel_strength(self):
+        df = load_steel_strength()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+
+    # Skip for circleCI efficiency
+    @unittest.skip
+    def test_load_jarvis_ml_dft_training(self):
+        df = load_jarvis_ml_dft_training()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df), 24759)
+        df = load_jarvis_ml_dft_training(drop_nan_columns='e mass_x')
+        self.assertEqual(len(df), 19978)
+
+    def test_load_jarvis_dft_2d(self):
+        df = load_jarvis_dft_2d()
+        self.assertTrue(isinstance(df, pd.DataFrame))
+        self.assertEqual(len(df), 636)
+        df = load_jarvis_dft_2d(drop_nan_columns='epsilon_x opt')
+        self.assertEqual(len(df), 522)
+
+    # Skip for circleCI efficiency
+    @unittest.skip
```

Comment:
Could you use the "skipIf" function of unittests to have this one run when not on a CI tool. I think `https://discuss.circleci.com/t/skip-certain-tests-in-python-nosetests/14001` shows how to check if you are on a CI. Never tried this, so I'm not sure if my suggestions are possible.

Reply:
I implemented this change, and it runs just fine locally, but it seems CircleCI is now failing to run the tests. I don't know if this is an issue caused by my update or something on the CIrcleCI side of things...

Reply:
Update: It was an issue with how the tox environment was set up and the checking of environment variables. It's now been resolved